src/System.Private.ServiceModel/tests/Common/Scenarios/ServiceInterfaces.cs

```diff
@@ -502,3 +502,13 @@ public interface IWcfAspNetCompatibleService
     [OperationContract(Action = "http://tempuri.org/IWcfService/EchoTimeAndSetCookie", ReplyAction = "http://tempuri.org/IIWcfService/EchoTimeAndSetCookieResponse")]
     string EchoTimeAndSetCookie(string name);
 }
+
+[ServiceContract]
+public interface IUtil
+{
+    [OperationContract]
+    byte[] GetClientCert(bool exportAsPem);
```

Comment:
NIT: spell out name, e.g. `GetMachineClientCertificate` or `GetUserClientCertificate` (there are two types of client certs we may need to use)